.circleci/config.yml

```diff
@@ -63,12 +63,18 @@ commands:
             bash ./Miniconda3-py38_4.8.3-Linux-x86_64.sh -b
             ~/miniconda3/bin/conda init bash
       - run:
-          name: Preparing environment
+          name: Preparing environment - Other dependency
           command: |
             sudo apt-get update
             sudo apt-get install -y expect fish openjdk-11-jre rsync
             conda create -n hydra python=<< parameters.py_version >> -yq
             conda run -n hydra pip install nox
+      - run:
+          name: Preparing environment - Hydra
+          command: |
+            conda create -n hydra python=<< parameters.py_version >> -yq
+            conda run -n hydra pip install nox
```

Comment:
You are starting to pile unrelated changes on this PR.

Reply:
this is actually needed.. I put some details  `4. install conda in circleCI linux docker` in the PR description on top on why this is needed.

Reply:
I said unrelated, not unneeded.
I am fine with us being consistent and using miniconda on Linux as well, but this can and should be in a different diff.

Reply:
got it. I will create a separate diff for this.